src/Controls/tests/DeviceTests/Elements/CollectionView/CollectionViewTests.Android.cs

```diff
@@ -144,6 +144,36 @@ await InvokeOnMainThreadAsync(() =>
 			});
 		}
 
+		[Fact(DisplayName = "CollectionView with SelectionMode None should not have click listeners")]
+		public void SelectionModeNoneDoesNotSetClickListeners()
+		{
+			SetupBuilder();
+
+			var collectionView = new CollectionView
+			{
+				ItemsSource = new[] { "Item 1", "Item 2", "Item 3" },
+				SelectionMode = SelectionMode.None
```

Comment:
Could create another test where SelectionMode changes at runtime?

Form Single to None for example.